### ISSUE TITLE
Pass in HTML5 validation props?

### ISSUE THREAD

**Issue by gtwilliams03** [schneidmaster/react-bootstrap-maskedinput#13]
Love this component and it has been running fine on my forgot password page for quite some time.  I recently updated to using the base HTML5 vlidation props (required, isValid, isRequired) which react-bootstrap's FormControl supports in their latest release for Bootstrap 4.  But I don't see the validation feedback when I enter text

```
<MaskedFormControl autoFocus autoComplete="off" 
    name="cellPhone" type='text' mask='(111) 111-1111' 
    value={cellPhone}                                         
    required 
    isValid={this.isValidated()}
    isInvalid={!this.isValidated()}
    onChange={(e) => this.handleCellPhoneChange(e)} />
```

![capture](https://user-images.githubusercontent.com/1564628/52793634-2943f000-303c-11e9-8bec-1eefeea8279a.JPG)

I do see this using react-bootstrap's@next FormControl (not using `MaskedFormControl`:

![capture2](https://user-images.githubusercontent.com/1564628/52793701-41b40a80-303c-11e9-8308-7fc2cf40b650.JPG)

Is this tied to an earlier version of react-bootstrap perhaps?

**Comment by schneidmaster** [comment]
Is this on `react-bootstrap` 1.0 beta? There is a breaking change that I will have to update `react-bootstrap-maskedinput` to account for (they renamed the `componentClass` prop to `as`) but I'm waiting for a stable version to land.

You should also look at the [source](https://github.com/schneidmaster/react-bootstrap-maskedinput/blob/master/src/index.js) for this package -- it is extremely simple. I have not kept up on `react-bootstrap` but it is possible that the validation feedback just will not work when you aren't using the default `FormControl` component class (i.e. when using this package which renders with `MaskedInput` instead). If there's a way this package could be improved to account for this use case I'm happy to accept a PR.

**Comment by gtwilliams03** [comment]
Yes - I just moved up to react-bootstrap 1.0 beta and you are correct on all counts.  `componentClass` has been replaced with `as` and the reason for my question is that the `validationState` prop on `FormGroup` has apparently gone by the wayside in favor of simply using the HTML5 validation props.  Your component works fine with react-bootstrap 1.0 beta - it just doesn't seem to show the validation props.  

I took at look at doing a PR but I can't figure out why the props aren't getting passed to the base `FormControl` component - it seems like its just a passthrough.
